Purchases/Public/RCPurchases.h

```diff
@@ -171,7 +171,8 @@ NS_SWIFT_NAME(Purchases)
 
 #pragma mark Configuration
 
-/** Set this to true if you are passing in an appUserID but it is anonymous, this is true by default if you didn't pass an appUserID
+/**
+ * TODO: Set this to true if you are passing in an appUserID but it is anonymous, this is true by default if you didn't pass an appUserID
```

Comment:
I think it should be true if using an anonymous ID. I'm not sure that we need to guide users beyond that.